tests/integration/pilot/sidecar_api_test.go

```diff
@@ -169,5 +149,5 @@ func validateMongoListener(t *testing.T, response *structpath.Instance) {
 // - Do cleanup before exit
 // - process testing specific flags
 func TestMain(m *testing.M) {
-	// framework.Main("sidecar_api_test", m, istio.SetupOnKube(&ist, nil))
+	framework.Main("sidecar_api_test", m, istio.SetupOnKube(&ist, nil))
```

Comment:
You can remove istio.SetupOnKube part for the time being. The tests are already tagged as native only at this point.